editor.planx.uk/src/@planx/components/FindProperty/Public/index.tsx

```diff
@@ -300,153 +201,50 @@ function GetAddress(props: {
   };
 
   return (
-    <Card
-      handleSubmit={() => props.setAddress(selectedOption)}
-      isValid={Boolean(selectedOption)}
-    >
-      <QuestionHeader
-        title={props.title || DEFAULT_TITLE}
-        description={props.description || ""}
-      />
-      <AutocompleteWrapper>
-        <InputLabel label="Postcode" htmlFor="postcode-input">
-          <Input
-            required
-            bordered
-            name="postcode"
-            id="postcode-input"
-            value={postcode || ""}
-            errorMessage={
+    <AutocompleteWrapper>
+      <InputLabel label="Postcode" htmlFor="postcode-input">
+        <Input
```

Comment:
future extra credit: replace this input with our `<postcode-search />` web component [over here](https://github.com/theopensystemslab/map/tree/main/src/components/postcode-search), which will further simplify this file and let us drop `postcode` package dependcy from editor. I expect the web component may need some styling overrides first though and double-checks that we have feature parity with especially change/keyUp/blur events and error handling.